### ISSUE TITLE
"__init__() got an unexpected keyword argument 'profile'" When attempting to connect to USB printer

### ISSUE THREAD

**Issue by tt2468** [python-escpos/python-escpos#447]
I have:
- [x] searched open and closed issues for duplicates

### Bug description
When I attempt to connect to a USB printer via `escpos.printer.Usb(0x04b8, 0x0202, 0, profile="TM-T88V")`, the `profile` argument claims to be unexpected. Removing the argument works without error.

### Steps to reproduce
- `sudo -H python3.7`
```python
from escpos.printer import Usb
p = Usb(0x04b8, 0x0202, 0, profile="TM-T88V")
```

### Device info

 **Printer:** Epson TM-T88V
 **SBC:** Raspberry Pi v3b

 **python-escpos version:** 2.2.0

 **python version:** 3.7.3

 **operating system:** `Raspbian GNU/Linux 10 (buster)`

**Comment by tt2468** [comment]
Fixed by upgrading with `--pre` in pip, via #317 . Crazy to me that this is still an issue.

**Comment by bxm6306** [comment]
Upgrading pip package with --pre didn't fix the issue for me on my Raspberry Pi 3b but simply leaving out the profile="TM-T88V" parameter did the trick as per your description, thanks!
